src/views/htmlcontent/src/js/components/app.component.ts

```diff
@@ -37,12 +37,18 @@ export interface IGridDataSet {
 // tslint:disable:max-line-length
 const template = `
 <div class="fullsize vertBox">
-    <div *ngIf="dataSets.length > 0" id="resultspane" class="boxRow header collapsible" [class.collapsed]="!resultActive" (click)="toggleResultsPane()">
+    <div *ngIf="dataSets.length > 0" role="button" id="resultspane" tabIndex="0" class="boxRow header collapsible"
```

Comment:
why not just use button element? it will handle the keypress as well.

Reply:
I'm not sure if angular supports these directives with buttons. We have it all styled, the only reason I put the role as a button was to support `aria-expanded`

Reply:
It's the same reason it's not a button in ADS either

Reply:
I do agree there are some styles need to be adjust if you change it to button, it is ok if you want to minimize the changes. As i pointed out earlier, you should handle the keypress event as well to treat space key and enter key as click.

Reply:
@alanrenmsft done